src/patternfly/components/DataList/data-list.scss

```diff
@@ -98,9 +98,8 @@
   --#{$data-list}__item-action--MarginInlineStart: var(--pf-t--global--spacer--md);
   --#{$data-list}__item-action--md--MarginInlineStart: var(--pf-t--global--spacer--xl);
   --#{$data-list}__item-action--not-last-child--MarginInlineEnd: var(--pf-t--global--spacer--md);
-  --#{$data-list}__item-action__action--MarginBlockStart: calc(var(--pf-t--global--spacer--sm) * -1); // former form-element
-  --#{$data-list}__action--MarginBlockStart: var(--#{$data-list}__item-action__action--MarginBlockStart); // update at breaking change
-  --#{$data-list}__item-action__action--MarginBlockEnd: calc(var(--pf-t--global--spacer--sm) * -1); // former form-element
+  --#{$data-list}__item-action__action--MarginBlockStart: calc(var(--pf-t--global--spacer--control--vertical--plain) * -1);
+  --#{$data-list}__item-action__action--MarginBlockEnd: calc(var(--pf-t--global--spacer--control--vertical--plain) * -1);
```

Comment:
Looks like `--#{$data-list}__action--MarginBlockStart` is set in the compact variant on line 155 but never used.

Reply:
(I think you can remove line 155?)